ProjectMarsLanguageFeature/Features/Language.feature

```diff
@@ -28,31 +28,31 @@ Scenario Outline:2. Add new languages with valid input
 Scenario: 3.Add new language with invalid input
 	Given Language Tab is selected in Profile Page/
 	Given Add New button should be there
-	 When I click on Add New buttons to give invalid input
-	 When I give space as input <'language'>,<'level'> for language
-	 Then <'language'> should not add
+	When I click on Add New buttons to give invalid input
```

Comment:
have you done destructive test cases?

Reply:
please also do this for skills

Reply:
I considered it as Long input given in example, may be  I need to give it separate and longer